android/src/main/java/com/github/wuxudong/rncharts/charts/BarLineChartBaseManager.java

```diff
@@ -62,6 +62,16 @@ public void setAutoScaleMinMaxEnabled(BarLineChartBase chart, boolean enabled) {
         chart.setAutoScaleMinMaxEnabled(enabled);
     }
 
+    @ReactProp(name = "visibleXRangeMinimum")
```

Comment:
what do you think of combining these and expose as a single "visibleXRange" property in js like:
```
{:visibleXRange {:min 1 :max 2}}
```
The benefit is that if these need to be updated frequently (e.g. inside a event handling), it could help reduce a round trip from js->native because the change is detected as a single property change as opposed to two? Though react might doing smart batching to avoid this but just in case it helps.

Reply:
Good thoughts, and it could help if we adjust the min/max, but going to punt on this for now, since not really necessary (we set the min to a constant and never alter it again). This attribute just tells the chart how far in you can zoom into (12 month period) and out of (no max applied)